jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -28,24 +30,72 @@ class Catalogs(PluginTemplateMixin, ViewerSelectMixin):
     template_file = __file__, "catalogs.vue"
     catalog_items = List([]).tag(sync=True)
     catalog_selected = Unicode("").tag(sync=True)
+    from_file = Unicode().tag(sync=True)
+    from_file_message = Unicode().tag(sync=True)
     results_available = Bool(False).tag(sync=True)
     number_of_results = Int(0).tag(sync=True)
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        # TODO: Add more catalogs.
+
         self.catalog = SelectPluginComponent(self,
                                              items='catalog_items',
                                              selected='catalog_selected',
-                                             manual_options=['SDSS'])
+                                             manual_options=['SDSS', 'From File...'])
```

Comment:
Is the `...` really necessary?
```suggestion
                                             manual_options=['SDSS', 'From File'])
```

Reply:
I included the `...` because I believe this is a generally accepted UX pattern to expect a popup dialog ("Save" vs "Save As...").  Granted this then gets a little confusing when setting from the API which does not invoke the popup, but still does require that you set another traitlet.  We could have a separate label (shown in the dropdown) and value (actual traitlet string), but I'm not sure that is worth the overhead?

Reply:
Did https://github.com/spacetelescope/jdaviz/pull/1707#discussion_r992460516 not happen? I was under the impression that you were going to change "From File" to "Load".

Reply:
The button in the dialog was changed from "Select" to "Load".  @Jenneh agreed that the `...` is a common pattern that we should use.  We can discuss whether to separate the API string from the UI label when we expose the API for the plugin.